modules/mobile/app/controllers/mobile/v0/prescriptions_controller.rb

```diff
@@ -7,23 +7,70 @@ module V0
     class PrescriptionsController < ApplicationController
       before_action { authorize :mhv_prescriptions, :access? }
 
+      # rubocop:disable Metrics/MethodLength
       def index
-        resource = client.get_history_rxs
+        begin
+          resource = client.get_history_rxs
+
+          # Temporary logging for prescription bug investigation
+          resource.attributes.each do |p|
+            Rails.logger.info('MHV Prescription Response',
+                              user: @current_user.uuid,
+                              params: params, id: p[:prescription_id],
+                              prescription: p)
+          end
+        rescue => e
```

Comment:
This will probably work for certain types of errors, but it will not work for errors that are already being handled by the `ExceptionHandling` module. The exception handler catches errors and renders rather than re-raising, so any error captured deeper in the stack will not make it here. You can validate this by adding a breakpoint here and running the specs. The error cases in the specs will not hit the breakpoint. The only time I hit the breakpoint was on a test that was not fully configured and errors on a missing vcr cassette.

Reply:
I think most of the error checking being done on those specs are parameter validation and authorization which happens outside the upstream service the rescue is capturing. I was able to hit a breakpoint in the rescue on upstream tracking error on line `399` of the specs where the upstream service returns the 404 .